### PR TITLE
Merge Avatar comp

### PR DESCRIPTION
**Pull request by piyushsh25**
image
1. There are 4 sizes sm,md,lg,xlg that come with default rounded corners.
2. Avatars(icons) with text
3.  Curved, and square avatars.

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import url("../../css/main.css");
+
+/* Avatar CSS */
+html{
+    background-color: var(--color-background);
+}
```
Comment (Tanay-J) [inline components/avatar/avatar.css:6]: I suggest to remove these lines. Keep only component related stuff in this stylesheet.

Comment (piyushsh25) [reply components/avatar/avatar.css]: done! thanks.

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    border-radius: var(--size0);
+}
+/* rounded corners */
+.avatar.curved{
+    border-radius: var(--size8);
+}
+.avatar.icon{
+    background-color: var(--color-icon);
+    color: var(--color-black);
+}
```
Comment (Ronak-Mutha) [inline components/avatar/avatar.css:45]: One suggestion to add a line break at the EOF to get rid of this warning.

File: components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  }
+ .avatar.sm{
+     width: var(--size20);
+     height: var(--size20);
+     font-size: var(--size10);
+ } 
+ .avatar.md{
+    width:  var(--size25);
+    height: var(--size25);    
+    font-size: var(--size10);
```
Comment (Ronak-Mutha) [inline components/avatar/avatar.css:21]: I'll suggest here to add font-size to base avatar styling as it is repeating for smaller and medium sizes.

**Pull request merged**